examples/http-different-payload-types/http_different_payload_types.bal

```diff
@@ -10,48 +10,42 @@ http:Client clientEP = check new ("http://localhost:9091/backEndService");
 service /actionService on new http:Listener(9090) {
 
     resource function 'default messageUsage()
-            returns string|http:InternalServerError {
+            returns string|http:InternalServerError|error? {
```

Comment:
If we add `|error?` do we need this `http:InternalServerError`

Reply:
* `http:InternalServerError` is a record type which could represent HTTP Internal Server Error and it has response body also.

Reply:
Ah yes, Sorry didn't see the `return {body:bStream.message()}` part. :)